assets/xml/objects/object_kamejima.xml

```diff
@@ -1,81 +1,99 @@
-        <Limb Name="object_kamejima_Standardlimb_00E65C" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_0C" Offset="0xE65C" />
-        <Limb Name="object_kamejima_Standardlimb_00E668" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_0D" Offset="0xE668" />
-        <Limb Name="object_kamejima_Standardlimb_00E674" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_0E" Offset="0xE674" />
-        <Limb Name="object_kamejima_Standardlimb_00E680" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_0F" Offset="0xE680" />
-        <Limb Name="object_kamejima_Standardlimb_00E68C" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_10" Offset="0xE68C" />
-        <Limb Name="object_kamejima_Standardlimb_00E698" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_11" Offset="0xE698" />
-        <Limb Name="object_kamejima_Standardlimb_00E6A4" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_12" Offset="0xE6A4" />
-        <Limb Name="object_kamejima_Standardlimb_00E6B0" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_13" Offset="0xE6B0" />
-        <Limb Name="object_kamejima_Standardlimb_00E6BC" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_14" Offset="0xE6BC" />
-        <Limb Name="object_kamejima_Standardlimb_00E6C8" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_15" Offset="0xE6C8" />
-        <Limb Name="object_kamejima_Standardlimb_00E6D4" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_16" Offset="0xE6D4" />
-        <Limb Name="object_kamejima_Standardlimb_00E6E0" Type="Standard" EnumName="OBJECT_KAMEJIMA_LIMB_17" Offset="0xE6E0" />
-        <Skeleton Name="object_kamejima_Skel_00E748" Type="Flex" LimbType="Standard" LimbNone="OBJECT_KAMEJIMA_LIMB_NONE" LimbMax="OBJECT_KAMEJIMA_LIMB_MAX" EnumName="ObjectKamejimaLimb" Offset="0xE748" />
-        <Animation Name="object_kamejima_Anim_00EF98" Offset="0xEF98" />
-        <Animation Name="object_kamejima_Anim_0100CC" Offset="0x100CC" />
-        <Animation Name="object_kamejima_Anim_010918" Offset="0x10918" />
-        <Animation Name="object_kamejima_Anim_0119D4" Offset="0x119D4" />
-        <Animation Name="object_kamejima_Anim_012260" Offset="0x12260" />
-        <Animation Name="object_kamejima_Anim_013264" Offset="0x13264" />
-        <Animation Name="object_kamejima_Anim_014E8C" Offset="0x14E8C" />
+        <DList Name="gTurtleWaveDL" Offset="0x140" />
+        <Texture Name="gTurtleWaveUnusedTex" OutName="turtle_wave_unused" Format="i8" Width="32" Height="64" Offset="0x2D8" />
+        <Texture Name="gTurtleWaveTex" OutName="turtle_wave" Format="i8" Width="64" Height="64" Offset="0xAD8" />
+        <TextureAnimation Name="gTurtleWaveTexAnim" Offset="0x1AF0" />
+
+
+        <!-- Turtle -->
+        <Collision Name="gTurtleAsleepCol" Offset="0x2328" />
+        <TextureAnimation Name="gTurtleEmptyTexAnim" Offset="0x2360"/>
+        <Collision Name="gTurtleAwakeCol" Offset="0x2470" />
+        <Collision Name="gTurtleUnusedCol" Offset="0x2E04" />
+        
+        <Animation Name="gTurtleHide1Anim" Offset="0x3980" />
```

Comment:
I wondered why there were so many "hide" animations, so I took a look into this object, and I think you're being bitten by some relatively undocumented changes in Z64Utils. In Random06457's original version of Z64Utils, trying to load these animations will throw an exception:
![image](https://user-images.githubusercontent.com/12245827/191217379-0bc31012-1df6-4fc8-b02f-87cf79722154.png)

However, in both Skawo's fork and my fork, these animations will instead show the skeleton in its default "folded" state; it just so happens that in the Turtle's case, this kind of looks like he's hiding. I ran into a similar thing with `object_az`. Loading the animation in-game shows that it *does* do something, even if it's very glitchy:

https://user-images.githubusercontent.com/12245827/191217877-c111fc2f-497e-4639-a35d-9bc2a9f81d62.mp4

So in short, these probably aren't hiding animations, and you'll probably need to just to view them in-game to document them. I'll add a warning in my Z64Utils fork for when this "folded state override" occurs.

Reply:
Ah I did not know that was the fallback behavior on errors. Thanks for the heads up. I did go and give them some different names.

Reply:
https://user-images.githubusercontent.com/31974442/191390104-ebc23015-e671-4279-b2d0-94a8e2c2441c.mp4

For reference the turtle is not having a very good day.

Reply:
While I knew they would probably be glitchy, I certainly didn't expect that, lol